From f25d2d82adfc4e62de3a942a9998640420e66b61 Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Mon, 16 Jan 2017 21:41:08 +0100
Subject: [PATCH 1549/2241] arm64: defconfig: enable PCA interrupt support in
 mvebu_v8_lsp_defconfig

The PCA9555 io-expander is part of armada-8040-modular and
armada-7040-modular board. Interrupt from io-expander are used for
various purposes e.g. MMC card-detect or usb device detect, therefore
enable support for it.

Change-Id: I15b01296a4af20e35e665b689b2aa53a986b61ed
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/38531
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index 2f964fc..fb05575 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -163,6 +163,7 @@ CONFIG_GPIO_MVEBU_A3700=y
 CONFIG_GPIO_PL061=y
 CONFIG_GPIO_XGENE=y
 CONFIG_GPIO_PCA953X=y
+CONFIG_GPIO_PCA953X_IRQ=y
 CONFIG_POWER_RESET_VEXPRESS=y
 CONFIG_POWER_RESET_XGENE=y
 CONFIG_POWER_RESET_SYSCON=y
-- 
2.7.4

